unity-renderer/Assets/Rendering/Culling/Tests/CullingControllerAndScenesShould.cs

```diff
@@ -63,73 +63,44 @@ public IEnumerator CullMovingEntities()
 
             yield return boxShape.routine;
 
-            yield return
-                new DCL.WaitUntil(() => entity.meshesInfo.renderers[0].forceRenderingOff, 0.3f);
-
+            Environment.i.platform.cullingController.objectsTracker.ForcePopulateRenderersList();
```

Comment:
these kind of singleton operations in the tests feels so wrong to me. We should define a refactor in the architecture to avoid this. Of course, out of the scope of this pr

Reply:
Agree, a later task about adding new tests should improve on this.